packages/anyclick-devtools/src/InspectDialog/ElementHierarchyNav.tsx

```
@@ -149,104 +226,251 @@ export function isAnyclickOwnedUI(element: Element): boolean {
+          break;
+        case "ArrowUp":
+          e.preventDefault();
+          setFocusedIndex((prev) => {
+            const next = prev - 1;
+            return next < 0 ? ancestors.length - 1 : next;
+          });
+          break;
+        case "Enter":
+        case " ":
+          e.preventDefault();
+          const focusedAncestor = ancestors[focusedIndex];
```

Biome (2.5.8)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)